backend/uclapi/oauth/views.py

```diff
@@ -94,17 +94,17 @@ def shibcallback(request):
     try:
         # Expire our signed tokens after five minutes for added security
         appdata = signer.unsign(appdata_signed, max_age=300)
-    except signing.BadSignature:
+    except signing.SignatureExpired:
         response = PrettyJsonResponse({
             "ok": False,
-            "error": "Bad signature. Please try login again."
+            "error": "Signature has expired. Please try login again."
```

Comment:
"Please try to login again" reads better. This is definitely my fault from when I wrote all this a long time ago, but whilst we're cleaning this up we might as well fix grammar too :P

In fact, since this is user facing, it might actually make more sense to write something like:
```
Login data has expired. Please attempt to log in again. If the issues persist please contact the UCL API Team to rectify this.
```